spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -1,3 +1,4 @@
+require 'ostruct'
```

Comment:
I’ve seen this started failing in other prs, too.
I haven’t had time to have a cloaer look.
If  we include it in our specs, would’t this cause failurea in user projects’ specs?

Ok to fix this in a separate pr?

Reply:
@pirj thank you. Yes, I've created #2754 to hopefully fix the error. It has been failing elsewhere. More details in that PR.